tests/mail/tests.py

```diff
@@ -319,6 +319,16 @@ def test_safe_mime_multipart(self):
         self.assertEqual(msg.message()['To'], '=?iso-8859-1?q?S=FCrname=2C_Firstname?= <to@example.com>')
         self.assertEqual(msg.message()['Subject'], '=?iso-8859-1?q?Message_from_Firstname_S=FCrname?=')
 
+    def test_safe_mime_multipart_with_attachments(self):
+        """
+        Make sure EmailMultiAlternatives include alternatives if the body is empty and it has attachments
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Make sure" since all tests do that. Please wrap docstrings and comments at 79 characters.